worker/src/RTC/SeqManager.cpp

```diff
@@ -1,6 +1,7 @@
 #define MS_CLASS "RTC::SeqManager"
 // #define MS_LOG_DEV_LEVEL 3
 
+#include <iterator>
```

Comment:
This must be at the end. Otherwise `make lint` will fail. `make format` will fix it for you.

Reply:
Will do myself, don't worry.